src/main/java/com/example/demo/domain/Member.java

```diff
@@ -1,20 +1,45 @@
 package com.example.demo.domain;
 
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+import static lombok.AccessLevel.PROTECTED;
+
+@Entity
+@Getter
+@Table(name = "member")
+@NoArgsConstructor(access = PROTECTED)
+@AllArgsConstructor
 public class Member {
 
+    @Id @GeneratedValue(strategy = IDENTITY)
```

Comment:
저는 어노테이션 떨어트려놓는게 더 보기 좋더라구요 ㅎ.ㅎ.....
```suggestion
    @Id 
    @GeneratedValue(strategy = IDENTITY)
```